src/messages/SharedMessageBuilder.cpp

```diff
@@ -150,7 +150,22 @@ void SharedMessageBuilder::parseHighlights()
     auto currentUser = getIApp()->getAccounts()->twitch.getCurrent();
     if (this->ircMessage->nick() == currentUser->getUserName())
     {
-        // Do nothing. We ignore any potential highlights from the logged in user
+        bool selfMessagesHighlights =
```

Comment:
This should be handled in the HighlightController. It will make more sense once you've merged https://github.com/Chatterino/chatterino2/pull/3835 has been merged into your branch

Reply:
Done in [d31cc6f](https://github.com/Chatterino/chatterino2/pull/3833/commits/d31cc6f182264daad530eac70df7c09eead779be)